WebExample/tests/input.spec.ts

```diff
@@ -0,0 +1,31 @@
+import {test, expect} from '@playwright/test';
+import * as TEST_CONST from '../../testConstants';
+import {checkCursorPosition, setupInput} from './utils';
+
+test.beforeEach(async ({page}) => {
+  await page.goto(TEST_CONST.LOCAL_URL, {waitUntil: 'load'});
+});
+
+test.describe('standard input behaviour', () => {
+  test('standard input results', async ({page}) => {
+    const inputLocator = await setupInput(page, 'clear');
+
+    await inputLocator.pressSequentially(TEST_CONST.EXAMPLE_CONTENT);
+    const value = await inputLocator.innerText();
+    expect(value).toEqual(TEST_CONST.EXAMPLE_CONTENT);
+  });
+
+  test('fast type cursor position', async ({page}) => {
```

Comment:
```suggestion
  test('long text, async ({page}) => {
```

Reply:
I'm not convinced - it doesn't describe this test properly. The main purpose was to verify if the cursor position would get desynchronized when typing fast (there was a bug with this behavior). It's not a standard long text input scenario.